src/components/Description/DescrAndButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import { Button } from "../Button";
+import { Description } from "../Description/Description";
+
+interface IProps {
+  isLogged?: boolean;
+}
+
+export const DescAndButton: FC<IProps> = ({ isLogged }) => (
+  <div>
+    {isLogged ? (
```

Comment:
as we are repeating a lot of the same things, we could have a variable object according to the `isLogged` like
```
const { descriptionText, buttonText } = {
        descriptionText: isLogged
          ? "Pri viacmesačnom predpladtnom zľava až 20%"
          : "Vyskúšajte si CHAT. Registrujte sa a získajte prvú hodinu zdarma.",
        buttonText: isLogged ? "Nákup kreditov" : "Registrovať sa"
      };
```
^ and then just use the variables `descriptionText` and `buttonText` in the render